services/clusters-keeper/src/simcore_service_clusters_keeper/core/settings.py

```diff
@@ -67,6 +67,11 @@ class WorkersEC2InstancesSettings(BaseCustomSettings):
         "this is required to start a new EC2 instance",
     )
 
+    WORKERS_EC2_INSTANCES_TIME_BEFORE_TERMINATION: datetime.timedelta = Field(
+        default=datetime.timedelta(minutes=3),
```

Comment:
MINOR: i know there are different formats that the pydantic parser can consume to produce `timedelta`. Can you add an example of value somewhere. E.g. in the example parameter

Reply:
fixed all the settings that are using timedelta, and puts a reference to the pydantic page for description.